client/src/interface/Contest.ts

```diff
@@ -29,6 +27,7 @@ export interface Contest {
       _id: string;
       files: string[];
       creator: string;
+      is_winner: boolean;
```

Comment:
Not a critical issue however, in JavaScript it is convention to use `camelCase` as opposed to `snake_case`